src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -41,9 +56,16 @@ protected void updateItem(Person person, boolean empty) {
                 setGraphic(null);
                 setText(null);
             } else {
-                setGraphic(new PersonCard(person, getIndex() + 1).getRoot());
+                PersonCard personCard = new PersonCard(person, getIndex() + 1);
```

Comment:
Similarly maybe can return in the above if branch so that you can unnest this else.